lib/src/data/simulations.dart

```diff
@@ -25,6 +26,7 @@ class Simulations with ChangeNotifier {
     4: "fourier series mathematics ",
     5: "lissajous curve pattern mathematics animation ",
     6: "epicycloid curve pattern mathematics animation pencil lines ",
+    7: "moure rose pattern"
```

Comment:
@kmrakash, just scanned this for now and I noticed that the spelling is wrong at a _lot_ of places in the PR, so you should probably fix that and keep it in mind for next time. It's "Maurer". 

Great effort, by the way!